lib/components/DataTable.js

```diff
@@ -2,12 +2,16 @@ import React, {PropTypes} from 'react';
 
 export default class Admin extends React.Component {
   static propTypes = {
-    data: PropTypes.arrayOf(PropTypes.object).isRequired,
+    data: PropTypes.arrayOf(PropTypes.object),
     showHeaders: PropTypes.bool
   }
 
   render() {
     const {data} = this.props;
+    if (!data || data.length < 1) {
```

Comment:
While eventually this whole thing should probably be redesigned, for the moment `componentWillReceiveProps` doesn't fire when the component mounts, so you'd have to also include it in `componentDidMount`. The current implementation is simpler.